index.dd

```diff
@@ -96,9 +96,9 @@ efficiency, control, and modeling power, with safety and programmer productivity
 
 $(SECTION3 Convenience,
 
-$(P D allows writing large code fragments without redundantly specifying types,
-like dynamic languages do. On the other hand, static inference deduces types and other
-code properties, giving the best of both the static and the
+$(P D allows you to write code without redundantly specifying types: it feels like you're writing
+in a dynamic language. But static type inference deduces types and other
+code properties, giving you the best of both the static and the
```

Comment:
"giving the best" is fine. The main concern in this paragraph was the ambiguity of the first sentence ("redundantly specifying types, like dynamic languages do").